frontend/src/component/admin/groups/GroupsList/GroupsList.tsx

```diff
@@ -134,6 +137,9 @@ export const GroupsList: VFC = () => {
                         <Grid key={group.id} item xs={12} md={6}>
                             <GroupCard
                                 group={group}
+                                rootRoles={roles.filter(
+                                    (role: IProjectRole) => role.type === 'root'
+                                )}
```

Comment:
You may want to extract stuff like this, even though this one is pretty simple.

```
// ...
const rootRoles = roles.filter(
    (role: IProjectRole) => role.type === 'root'
)

// ...

rootRoles={rootRoles}
```

Reply:
Yes! Thank you, nice suggestion!